.github/workflows/integration-test-upgrade-suite.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
     - name: TestCephUpgradeSuite
       run: |
        export DEVICE_FILTER=$(lsblk|awk '/14G/ {print $1}'| head -1)
-       go test -v -timeout 1800s -run CephUpgradeSuite github.com/rook/rook/tests/integration
+       go test -v -timeout 1800s -failfast -run CephUpgradeSuite github.com/rook/rook/tests/integration
```

Comment:
How fast do we expect the test to fail with this option? I see an upgrade test failed, but it doesn't look like it failed immediately either.

Reply:
It depends of the amount of "sub-tests" the test has, if none then it will take the same amount of time.

Reply:
Also the particular failure here is different since the job timed out:

`panic: test timed out after 30m0s`

Reply:
There was a test assert a few minutes before the test timed out, so I was assuming the fast failure would hit after the first assert, rather than waiting for a timeout. But anyway, let's try this change.